src/main/java/io/vertx/ext/jdbc/package-info.java

```diff
@@ -14,36 +14,101 @@
  * You may elect to redistribute this code under either of these licenses.
  */
 /**
- * = Vert.x JDBC Service
+ * = Vert.x JDBC service
  *
- * The {@link io.vertx.ext.jdbc.JdbcService JDBC Service} is responsible for providing an interface for Vert.x applications that need to interact with
- * a database using a JDBC driver.
+ * This service allows you to interact with any JDBC compliant database using an asynchronous API from your Vert.x
+ * application.
  *
- * == Getting Started
+ * == Setting up the service
  *
- * Since the JDBC service is a service proxy, there are several ways to get started, but the easiest is to just deploy the service
- * verticle:
- * [source,{lang}]
+ * As with other services you can use the service either by deploying it as a verticle somewhere on your network and
+ * interacting with it over the event bus, either directly by sending messages, or using a service proxy, e.g.
+ *
+ * Somewhere you deploy it:
+ *
+ * [source,java]
```

Comment:
The `[source,java]` part doesn't get docgen'd for groovy and javascript - I guess that's just not done in docgen yet?